pages/data-providers/[data-provider-id]/deposit-compliance.jsx

```diff
@@ -36,6 +37,33 @@ const SidebarContent = ({ context: { oai, originalId, authors, title } }) => {
   )
 }
 
+class PublicationDateColumn extends Table.Column {
+  render() {
+    const {
+      context: { publicReleaseDate, publicationDateValidationLevel },
+    } = this.props
+
+    return (
+      <span>
+        {dayjs(publicReleaseDate).format('DD/MM/YYYY')}
```

Comment:
Can we utilise Intl here too? (for the future)

Reply:
What do you mean? The format is the same as if you would use Intl

```
console.log(new Intl.DateTimeFormat('en-GB').format(date));
// expected output: "20/12/2012"
```

Reply:
That the only point. `Intl` is a standard and the `dayjs` is not